recipe/meta.yaml

```diff
@@ -15,7 +15,7 @@ source:
 build:
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 2
-  skip: True  # [win or osx]
+  skip: True  # [ppc64le and py==313]
```

Comment:
```suggestion
  skip: True  # [ppc64le and py>312]
```

Maybe just condition on if greater than Python 3.12? This way future builds for Python 3.14+ won't cause issues.

Reply:
Oh yes, that's much better, thank you!